alerts/resource_alerts.libsonnet

```diff
@@ -88,6 +88,19 @@
               message: 'Namespace {{ $labels.namespace }} is using {{ $value | humanizePercentage }} of its {{ $labels.resource }} quota.',
             },
           },
+          {
+            alert: 'KubeNodeEvictedPods',
+            expr: |||
+              increase(kubelet_evictions{%(kubeStateMetricsSelector)s}[3m]) > 0
```

Comment:
We want to make sure that the range selector selects at least three samples, so I'd suggest bumping this to `5m`.